README.md

```diff
@@ -24,9 +9,8 @@ to Java by the [aws-crt-java](https://github.com/awslabs/aws-crt-java) package.
 * [Samples](samples)
 * [Getting Help](#getting-help)
 * [FAQ](./documents/FAQ.md)
-* [Giving Feedback and Contributions](#giving-feedback-and-contributions)
+* [API Docs](https://aws.github.io/aws-iot-device-sdk-java-v2/)
 * [MQTT5 User Guide](./documents/MQTT5_Userguide.md)
-* [DEVELOPING](./documents/DEVELOPING.md)
 
```

Comment:
Would we like to keep the development instruction here?

Reply:
I think it will be best down with the getting help links. What are your thoughts?